frontend/src/scenes/authentication/signup/verify-email/VerifyEmail.tsx

```diff
@@ -51,12 +100,12 @@ export function VerifyEmail(): JSX.Element {
         <div className="flex h-full flex-col">
             <div className="flex h-full">
                 <BridgePage view="verifyEmail" fixedWidth={false}>
-                    <div className="px-12 py-8 text-center flex flex-col items-center max-w-200">
+                    <div className="px-12 py-8 text-center flex flex-col items-center max-w-160">
                         {view === 'pending' ? (
                             <>
                                 <h1 className="text-xl">Welcome to PostHog!</h1>
                                 <h1 className="text-3xl font-bold">Let's verify your email address.</h1>
-                                <div className="max-w-80 my-12">
+                                <div className="max-w-60 my-10">
                                     <MailHog className="w-full h-full" />
                                 </div>
                                 <p>
```

Comment:
Can't comment on line 112 but we could remove the "If you have not received the email in a few minutes, please check your spam folder." since it's in the help checklist.

Reply:
more nits: Could simplify `An email has been sent to with a link to verify your email address.` to `Click the link we sent to ${email}`. 

Also note that the original copy is missing words in the bolded section. Sent to ...? `An email has been sent **to with** a link to verify your email address.`